templates/ocs-deployment/PersistentVolumeClaim.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ pvc_name | default('ocsci-pvc') }}
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
I guess it would be better to make this jinja2 as well, so that same pvc.yaml can be used for both cephfs and rbd

Reply:
Removed PersistentVolumeClaim.yaml